client/task-list/tasks/payments/methods.js

```diff
@@ -119,9 +118,15 @@ export function getPaymentMethods( {
 			),
 			before: <WCPayIcon />,
 			visible:
-				[ 'US' ].includes( countryCode ) &&
-				! hasCbdIndustry &&
-				isJetpackConnected,
+				[
+					'US',
+					'AS',
+					'PR',
+					'VI',
+					'GU',
+					'MP',
+					'UM', // U.S. and its territories (which WooCommerce classifies as countries)
```

Comment:
Looks like only US and PR are supported, according to [this resource](https://support.stripe.com/questions/stripe-availability-for-outlying-territories-of-supported-countries).

Reply:
Great find! Thanks for double-checking. Fixed in https://github.com/woocommerce/woocommerce-admin/pull/4499/commits/fd1cafc9555967aa0f9ef0726d4e46bc41d5fa12